kentik_api_library/examples/queries_visualisation_example.ipynb

```diff
@@ -0,0 +1,373 @@
+       "      <td>35206.666667</td>\n",
+       "      <td>60</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>1610357640000</td>\n",
+       "      <td>22875.333333</td>\n",
+       "      <td>60</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>1610357700000</td>\n",
+       "      <td>23918.800000</td>\n",
+       "      <td>60</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>56</th>\n",
+       "      <td>1610360820000</td>\n",
+       "      <td>20324.533333</td>\n",
+       "      <td>60</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>57</th>\n",
+       "      <td>1610360880000</td>\n",
+       "      <td>26223.600000</td>\n",
+       "      <td>60</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>58</th>\n",
+       "      <td>1610360940000</td>\n",
+       "      <td>21310.800000</td>\n",
+       "      <td>60</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>59</th>\n",
+       "      <td>1610361000000</td>\n",
+       "      <td>16589.200000</td>\n",
+       "      <td>60</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>60</th>\n",
+       "      <td>1610361060000</td>\n",
+       "      <td>16940.400000</td>\n",
+       "      <td>60</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>61 rows × 3 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                0             1   2\n",
+       "0   1610357460000  16648.266667  60\n",
+       "1   1610357520000    226.400000  60\n",
+       "2   1610357580000  35206.666667  60\n",
+       "3   1610357640000  22875.333333  60\n",
+       "4   1610357700000  23918.800000  60\n",
+       "..            ...           ...  ..\n",
+       "56  1610360820000  20324.533333  60\n",
+       "57  1610360880000  26223.600000  60\n",
+       "58  1610360940000  21310.800000  60\n",
+       "59  1610361000000  16589.200000  60\n",
+       "60  1610361060000  16940.400000  60\n",
+       "\n",
+       "[61 rows x 3 columns]"
+      ]
+     },
+     "execution_count": 26,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.6"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```

Comment:
could you please also add a second pandas example that sorts by given column and takes for example top 10 rows?
This is something they were interested in :)